tine20/Admin/Frontend/Cli.php

```diff
@@ -209,7 +209,43 @@ public function deleteAccount($_opts)
 
         echo 'deleted account ' . $args['accountName'] . PHP_EOL;
     }
-
+    /**
+     * Delete containers with no users
+     *
+     * @return void
+     */
+    public function deleteUserlessContainers(Zend_Console_Getopt $opts)
+    {
+        if ($opts->d) {
+            echo "--DRY RUN--\n";
+        }
+        // Get an instance of Admin_Frontend_Json
+        $jsonFrontend = new Admin_Frontend_Json();
+
+        // Get all containers
+        $containers = $jsonFrontend->searchContainers([], null);
+
+        foreach ($containers['results'] as $container) {
+          // Check if the container has no users
+          $user_search = $jsonFrontend->searchUsers([
+              [
+                'field' => 'container_id',
+                'operator' => 'equals',
+                'value' => $container['id'],
+              ],
+            ], null);
+            if (empty($user_search['totalcount'])) {
```

Comment:
it won't work like this - we need to get all container "owner"s (= user id) and then search for existing users with this id. if there is no user -> delete the container:

~~~php
try {
  $user = Tinebase_User::getInstance()->getFullUserById($container['owner']);
} catch (Tinebase_Exception_NotFound) {
  $jsonFrontend->deleteContainers([$container['id']]);
}